techniques/general/G138.html

```diff
@@ -2,78 +2,77 @@
-            they relate to the normative WCAG 2.1 success criteria. The Applicability section
-            explains the scope of the technique, and the presence of techniques for a specific
-            technology does not imply that the technology can be used in all situations to create
-            content that meets WCAG 2.1.
+         <h2>達成方法に関する重要な情報</h2>
+         <p>これらの達成方法 (参考) の使用法及び、それらが WCAG 2.1 達成基準 (規定) とどのように関係するかに関する重要な情報については、<a href="https://waic.jp/docs/WCAG21/Understanding/understanding-techniques.html">WCAG 達成基準の達成方法を理解する</a>を参照のこと。適用 (対象) セクションは、その達成方法の範囲について説明しており、特定の技術に関する達成方法があるからといって、WCAG 2.1 を満たすコンテンツを作成する際に、常にその技術が使用可能であるわけではない。
          </p>
       </section>
       <main>
          <section id="applicability">
-            <h2>Applicability</h2>
-            <p>All technologies that support color and text.</p>
-            <p>This technique relates to <span><a href="https://w3c.github.io/wcag/understanding/info-and-relationships">Success Criterion 1.3.1: Info and Relationships</a> (Sufficient as a way to meet an unwritten technique)</span>.
+            <h2>適用 (対象)</h2>
+            <p>色及びテキストをサポートするウェブコンテンツ技術すべて。</p>
+            <p>これは<span><a href="https://w3c.github.io/wcag/understanding/info-and-relationships">達成基準 1.3.1: 情報及び関係性</a> (書かれていない達成方法を満たす慣習的な達成方法)</span> に関する達成方法である。
             </p>
          </section>
          <section id="description">
-            <h2>Description</h2>
-            <p>The objective of this technique is to combine color and semantic markup to convey
-               information. Most users can quickly scan the content to locate information conveyed
-               by using color. For users who cannot see color, semantic markup can provide a different
-               type of cue. User agents can then make this type of structure perceivable to the user,
-               for example using a different visual presentation for different types of structures
-               or by using a different voice or pitch in an auditory presentation.
+            <h2>解説</h2>
+            <p>この達成方法の目的は、色とセマンティックなマークアップを組み合わせて、情報を伝達することである。ほとんどの利用者はコンテンツをすばやく読み取り、色を用いて伝達された情報をコンテンツから見つけることができる。色を見ることができない利用者に対しては、セマンティックなマークアップによって異なるタイプの手がかりを提供することができる。それによって、ユーザエージェントは、例えば、異なるタイプの構造に対しては異なる視覚的提示を用いる、又は聴覚的提示において異なる音声もしくは音の高さを用いることによって、このタイプの構造を利用者に対して知覚可能にすることができる。
             </p>
-            <p>Most user agents will visually distinguish text that has been identified using semantic
-               markup. Some assistive technologies provide a mechanism for determining the characteristics
-               of content that has been created using proper semantic markup.
+            <p>ほとんどのユーザエージェントは、セマンティックなマークアップを用いて指定されたテキストを視覚的に区別する。支援技術の中には、適切なセマンティックマークアップを用いて制作されたコンテンツの特徴を定めるメカニズムを提供しているものもある。
```

Comment:
```suggestion
            <p>ほとんどのユーザエージェントは、セマンティックなマークアップを用いて指定されたテキストを視覚的に区別する。支援技術の中には、適切なセマンティックなマークアップを用いて制作されたコンテンツの特徴を定めるメカニズムを提供しているものもある。
```

Reply:
```suggestion
            <p>ほとんどのユーザエージェントは、セマンティックなマークアップを用いて指定されたテキストを視覚的に区別する。支援技術の中には、適切なセマンティックのマークアップを用いて制作されたコンテンツの特徴を定めるメカニズムを提供しているものもある。。
```

Reply:
「適切**な**セマンティック**な**マークアップ」と「な」が続くのが気になったので、微修正しました。
でも、直前のセンテンスの「セマンティック**な**マークアップを用いて...」に合わせると、もんどさん提案のほうがよいのかも...と正直迷っています。

Reply:
確かに「な」が2回出てくるのは微妙ですね…これはどうでしょう。

```suggestion
            <p>ほとんどのユーザエージェントは、セマンティックなマークアップを用いて指定されたテキストを視覚的に区別する。支援技術の中には、適切でセマンティックなマークアップを用いて制作されたコンテンツの特徴を定めるメカニズムを提供しているものもある。
```

Reply:
ありがとうございます。問題ないと思います！